samples/nrf_desktop/src/hw_interface/buttons.c

```diff
@@ -335,19 +366,31 @@ void button_pressed(struct device *gpio_dev, struct gpio_callback *cb,
 static void init_fn(void)
 {
 	/* Setup GPIO configuration */
-	gpio_dev = device_get_binding(DT_GPIO_P0_DEV_NAME);
-	if (!gpio_dev) {
-		LOG_ERR("Cannot get GPIO device binding");
-		return;
+	const char *gpio_dev_names[] = {DT_GPIO_P0_DEV_NAME,
+					DT_GPIO_P1_DEV_NAME};
+
+	for (size_t i = 0; i < GPIO_PORT_NUM; i++) {
+		if (port_used(i)) {
```

Comment:
sure it is used if port num is this high (the benefit of having it configured in the header along with the pins)